### PR TITLE
DOC-10466 Docs for o11y/console: Improve performance of the Databases Page for large clusters

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-10466

- In cockroachcloud/databases-page.md, modified to use v1 include files.
- Renamed include files databases.md and index-details.md to databases-v1.md and index-details-v1.md for v24.3 and copied both for v24.2.
- Created cockroachcloud/databases-page-v24.3.
- In ui-databases-page.md and cockroachcloud/databases-page-v24.3, modified to use new/updated include files.
- Updated include file index-recommendations.md.
- Created include files databases-list-page.md, database-details-page.md, table-details-page.md, index-details-page.md.
- In v24.3/performance-recipes.md and _includes/v24.3/ui/statement-details.md, updated links to ui-databases-page.md.

Rendered previews
- [DB Console Databases page](https://deploy-preview-19026--cockroachdb-docs.netlify.app/docs/v24.3/ui-databases-page.html)
- [Cloud Console Databases page](https://deploy-preview-19026--cockroachdb-docs.netlify.app/docs/cockroachcloud/databases-page-v24.3.html)

### CODE REVIEW

**Comment by florence-crl** [comment]
@xinhaoz and @kyle-a-wong please review this docs pr if possible within the week. thanks!

File: src/current/_includes/v24.3/ui/databases-list-page.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+On the [Databases List Page](#databases-list-page) and the [Database Tables Tab](#tables-list-tab) of the [Database Details Page](#database-details-page), above the table of information, the **Last refreshed** indicator displays how long ago the cached data was last refreshed automatically (per the cluster settings) or manually.
+
+The cached data can be refreshed in the following ways:
+
+- Automatically: Set the cluster setting `obs.tablemetadata.automatic_updates.enabled` to `true`. The frequency of refreshes is controlled by the cluster setting `obs.tablemetadata.data_valid_duration`. Automatic updates is disabled by default to avoid unnecessarily consuming resources. However, if you want databases and tables information immediately when you visit the corresponding page, then enable automatic updates.
+- Manually: When any user clicks the **Refresh data** icon button to the right of the **Last refreshed** indicator. If a refresh job has been triggered, the **Refresh data** icon button will be disabled and the hover text will display `Data is currently refreshing`.
```
Comment (xinhaoz) [inline src/current/_includes/v24.3/ui/databases-list-page.md:59]: Just FYI that this text will change from `Data is currently refreshing` to show the job's start time and the progress https://github.com/cockroachdb/cockroach/pull/133188. Release note there comments on this.

**Review by xinhaoz — APPROVED** [review]
Thank you Florence!

File: src/current/_includes/v24.3/ui/databases-list-page.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+The [cluster settings]({% link {{ version_prefix }}/cluster-settings.md %}) that control the refresh behaviour for the cached data in the `system.table_metadata` table are:
+
+- `obs.tablemetadata.automatic_updates.enabled` (default: `false`) enables automatic updates of the table metadata cache `system.table_metadata`.
+- `obs.tablemetadata.data_valid_duration` (default: `20m0s`) is the duration for which the data in `system.table_metadata` is considered valid.
+
+On the [Databases List Page](#databases-list-page) and the [Database Tables Tab](#tables-list-tab) of the [Database Details Page](#database-details-page), above the table of information, the **Last refreshed** indicator displays how long ago the cached data was last refreshed automatically (per the cluster settings) or manually.
+
+The cached data can be refreshed in the following ways:
+
+- Automatically: Set the cluster setting `obs.tablemetadata.automatic_updates.enabled` to `true`. The frequency of refreshes is controlled by the cluster setting `obs.tablemetadata.data_valid_duration`.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/_includes/v24.3/ui/databases-list-page.md]: updated

File: src/current/_includes/v24.3/ui/databases-list-page.md (hunk 1 above)
Comment (florence-crl) [reply src/current/_includes/v24.3/ui/databases-list-page.md]: updated

**Review by kyle-a-wong — APPROVED** [review]
Looks great, thanks Florence!

File: src/current/_includes/v24.3/ui/database-details-page.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+The following information is displayed for each table:
+
+ Column                        | Description
+-------------------------------|-------------
+Name                           | The name of the table. Click a table name to view the [Table Details Page](#table-details-page) for the selected table.
+Replication Size               | The approximate compressed total disk size across all replicas of the table.
+Ranges                         | The number of ranges in the table.
+Columns                        | The number of columns in the table.
+Indexes                        | The number of indexes in the table.
```
Comment (kathancox) [inline src/current/_includes/v24.3/ui/database-details-page.md:19]: do you want to link replicas, ranges, indexes here?

File: src/current/_includes/v24.3/ui/database-details-page.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+{% include_cached new-in.html version="v24.3" %} To filter databases based on the nodes on which the database tables are located, use the nodes multi-select dropdown above the list table:
+
+1. Click the dropdown arrow.
+1. Select one or more nodes. You may need to scroll down for nodes in different regions. You may also type in the beginning of the node name to narrow the list.
+
+    The list of tables is filtered by the nodes selected.
+
+{% if page.cloud == true  -%}
+NOTE: Nodes multi-select dropdown is not available on Standard or Basic clusters.
```
Comment (kathancox) [inline src/current/_includes/v24.3/ui/database-details-page.md:44]: Remove the caps here, and add a note box, or rephrase into a sentence.

**Review by kathancox — APPROVED** [review]
Couple nits, but otherwise LGTM. Where there are repeats of content I tried to add to all instances, but I may have missed some.

File: src/current/_includes/v24.3/ui/database-details-page.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+The following information is displayed for each table:
+
+ Column                        | Description
+-------------------------------|-------------
+Name                           | The name of the table. Click a table name to view the [Table Details Page](#table-details-page) for the selected table.
+Replication Size               | The approximate compressed total disk size across all replicas of the table.
+Ranges                         | The number of ranges in the table.
+Columns                        | The number of columns in the table.
+Indexes                        | The number of indexes in the table.
+Regions/Nodes                  | Regions/Nodes on which the table's data is stored.{% if page.cloud == true %}<br><br>NOTE: Not available on Standard or Basic clusters.{% endif %}
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/_includes/v24.3/ui/database-details-page.md]: updated

File: src/current/_includes/v24.3/ui/databases-list-page.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+### Refresh data
+
+{% include_cached new-in.html version="v24.3" %} The `system.table_metadata` table caches the necessary metadata of a table that populates the following:
+
+- [Databases List Page](#databases-list-page)
+- [Database Tables Tab](#tables-list-tab) of the [Database Details Page](#database-details-page)
+- [Overview Tab](#overview-tab) of the [Table Details Page](#table-details-page)
+
+The [cluster settings]({% link {{ version_prefix }}/cluster-settings.md %}) that control the refresh behaviour for the cached data in the `system.table_metadata` table are:
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/_includes/v24.3/ui/databases-list-page.md]: updated

File: src/current/cockroachcloud/databases-page-v24.3.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+The **Databases** page of the CockroachDB {{ site.data.products.cloud }} Console provides details of the following:
+
+- The databases configured.
+- The tables in each database and the indexes on each table.
+- The grants assigned to each role and user.
+- [Index recommendations](#index-recommendations).
+
+To view this information, select a cluster from the [**Clusters** page]({% link cockroachcloud/cluster-management.md %}#view-clusters-page), and click **Databases** in the **Data** section of the left side navigation menu. This will give you access to the following:
+
+- [Databases List Page](#databases-list-page): initial page which lists the databases on the cluster.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/cockroachcloud/databases-page-v24.3.md]: updated

File: src/current/v24.3/ui-databases-page.md [hunk 8]
```diff
@@ -14,10 +17,31 @@ The **Databases** page of the DB Console provides details of the following:
 - The grants assigned to each role and user.
 - [Index recommendations](#index-recommendations).
 
-To view this page, [access the DB Console]({% link {{ page.version.version }}/ui-overview.md %}#db-console-access) and click **Databases** in the left-hand navigation.
+To view this information, [access the DB Console]({% link {{ version_prefix }}/ui-overview.md %}#db-console-access) and click **Databases** in the left side navigation menu. This will give you access to the following:
+
+- [Databases List Page](#databases-list-page): initial page which lists the databases on the cluster.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/v24.3/ui-databases-page.md]: updated

**Review by florence-crl — COMMENTED** [review]
TFTRs!

**Pull request merged**